subcommands/version/cmd.go

```diff
@@ -10,23 +10,46 @@ import (
 )
 
 var Commit string
+var updateTo string
 
 func NewCommand() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "version",
 		Short: "Show version information of this tool.",
 		Run:   doVersion,
 	}
+	cmd.Flags().StringVarP(&updateTo, "update-to", "", "", "Update to the given version of fioctl")
+	return cmd
 }
 
 func doVersion(cmd *cobra.Command, args []string) {
 	fmt.Println(Commit)
 	updateFinder := newUpdateFinder(cmd)
 	update, err := updateFinder.FindLatest()
 	DieNotNil(err)
+
+	if len(updateTo) != 0 {
+		if updateTo != update.Version.String() {
+			DieNotNil(fmt.Errorf("invalid version %s != %s", updateTo, update.Version))
+		}
+		if err := update.Do(); err != nil {
+			if os.IsPermission(err) {
+				fmt.Println("There was a permission error while updating fioctl. Please run this again with 'sudo'")
```

Comment:
I'd still call `DieNotNil` in this case.

Reply:
c82d19703a95e9cfb5ddf875d240a5ebf360b65d

Reply:
Maybe, this would look better:
```
There was a permission error while updating fioctl: %w.\nPlease, run this again with 'sudo'.
```

I'm tempting to add `sudo !!` here, but that's probably too geeky.
Also, I guess this code path cannot occur on Windows where there is no `sudo`, right?
If not, we can paraphrase this to smth like `Please, run this command again as an admin`.

Reply:
9b1e12361ca1a48c32e033319dc47e85710962b0